tests/test_testing.py

```diff
@@ -192,3 +197,40 @@ def on_post(self, req, resp):
     result = testing.simulate_post(
         app, '/', json={}, headers=headers, content_type=falcon.MEDIA_HTML)
     assert result.text == falcon.MEDIA_JSON
+
+
+@pytest.mark.parametrize('cookies', [
+    {'foo': 'bar', 'baz': 'foo'},
+    CustomCookies()
+])
+def test_create_environ_cookies(cookies):
+    environ = testing.create_environ(cookies=cookies)
+
+    assert environ['HTTP_COOKIE'] in ('foo=bar; baz=foo', 'baz=foo; foo=bar')
+
+
+def test_create_environ_cookies_options_method():
+    environ = testing.create_environ(method='OPTIONS', cookies={'foo': 'bar'})
+
+    assert 'HTTP_COOKIE' not in environ
+
+
+def test_cookies_jar():
+    class Foo:
+        def on_get(self, req, resp):
+            resp.set_cookie('has_permission', 'true')
+
+        def on_post(self, req, resp):
+            if req.cookies['has_permission'] == 'true':
+                resp.status = falcon.HTTP_200
+            else:
+                resp.status = falcon.HTTP_403
+    app = App()
+    app.add_route('/jars', Foo())
+
+    client = testing.TestClient(app)
+
+    response_one = client.simulate_get('/jars')
+    response_two = client.simulate_post('/jars', cookies=response_one.cookies)
```

Comment:
It would be great to surface this pattern in the docs. At a minimum, we should mention in the `Result` docstring that the `cookies` property can be passed directly as the cookies kwarg when simulating subsequent requests.

Reply:
Yes, nice catch, I'll update the docstring.